src/menu_items.js

```diff
@@ -0,0 +1,63 @@
+class MenuItems {
+
+ static  showItems(){
+
+       let items = {
+            "menuList": [
+    
+                {
+                    "Name": "Italian breakFirst",
+                    "price": 185,
+                    "image": "../assets/images/1.jpg"
+                },
+                {
+                    "Name": "Desert",
+                    "price": 200,
+                    "image": "../assets/images/2.jpg"
+                },
+                {
+                    "Name": "lunch meal",
+                    "price": 150,
+                    "image": "../assets/images/4.jpg"
+                },
+                {
+                    "Name": "Chinese meal",
+                    "price": 400,
+                    "image": "../assets/images/5.jpg"
+                },
+                {
+                    "Name": "Chinese meal",
+                    "price": 400,
+                    "image": "../assets/images/5.jpg"
+                },
+                
+                {
+                    "Name": "Chinese meal",
+                    "price": 400,
+                    "image": "../assets/images/5.jpg"
+                },
+                {
+                    "Name": "Chinese meal",
+                    "price": 400,
+                    "image": "../assets/images/5.jpg"
+                },
+                {
+                    "Name": "Chinese meal",
+                    "price": 400,
+                    "image": "../assets/images/5.jpg"
+                },
+               
+    
+            ]
+        }
+
+        return items;
+    }
+
+    
+
+}
+
+export {
+    MenuItems
+};
```

Comment:
Kindly arrange your code properly, make it more professional by using the one line space and work on the indentation. 👍 
do the same to all files.
see below;
![image](https://user-images.githubusercontent.com/31897434/89600197-5bad6280-d859-11ea-8c5f-a0c8e1a81b10.png)